app/org/sagebionetworks/bridge/stormpath/StormpathAccountDao.java

```diff
@@ -492,12 +508,24 @@ private Account constructAccount(StudyIdentifier studyId, com.stormpath.sdk.acco
         return account;
     }
     
-    private Account getAccountWithEmail(Study study, String email) {
+    public String getAccountIdFromEmail(Study study, String email) {
+        Directory directory = client.getResource(study.getStormpathHref(), Directory.class);
```

Comment:
Yes you're right, this must have been part of an earlier attempt to implement. Removed.